src/Configurations/ListPane/List.js

```diff
@@ -23,16 +22,20 @@ export const List = props => {
 
   const query = Configurations.useListQuery();
 
-  const localeDateFormat = useMemo(
-    () => moment.localeData(stripes.locale).longDateFormat('L'),
-    [stripes.locale],
-  );
+  const getFormattedLastUpdate = (value, locale) => Intl.DateTimeFormat(locale, {
```

Comment:
You can use useIntl hook which is already used here. That will be like `intl.formatDate` instead of `Intl.DateTimeFormat`

Reply:
Or `getFormattedLastUpdate` could be made a standalone function, not in the component body

Reply:
> You can use useIntl hook which is already used here. That will be like `intl.formatDate` instead of `Intl.DateTimeFormat`

`Intl` here is not the `intl` of react-intl
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

Reply:
> > You can use useIntl hook which is already used here. That will be like `intl.formatDate` instead of `Intl.DateTimeFormat`
> 
> `Intl` here is not the `intl` of react-intl
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

Yes, based on previos inplemenation with 'moment' we should convert Date to UTC + format date based on locale, intl.formatDate hook's method doesn't support 'locale' param, but we need it. There is a reason why I've used "Intl.DateTimeFormat"

![image](https://user-images.githubusercontent.com/86330150/125072419-42451400-e0c3-11eb-8973-1ea6104004ef.png)

There is 'localeMatcher' prop only, I'm not sure that is what we are looking for. Didn't find it the documentation.

Reply:
> Yes, based on previos inplemenation with 'moment' we should convert Date to UTC + format date based on locale, intl.formatDate hook's method doesn't support 'locale' param, but we need it. There is a reason why I've used "Intl.DateTimeFormat"

Why do you need a `locale` param in this case? It seems like you're always just passing in the locale stored in Stripes here, and I'm under the impression that react-intl will by default use that locale because we instantiated it to use that locale.